make_prg/__init__.py

```diff
@@ -4,6 +4,6 @@
 try:
     __version__ = get_distribution("make_prg").version
 except:
-    __version__ = "1.0.0"
+    __version__ = "0.4.0"
```

Comment:
I don't like this; we now have the version in two places.

See https://github.com/snakemake/snakefmt/blob/master/snakefmt/__init__.py for how we handle version in snakefmt. We use pyproject.toml instead of setup.py, but it *should* work the same (I haven't tested)